src/test/e2e/create-place.spec.ts

```diff
@@ -51,9 +51,10 @@ test('create a place', async ({ baseURL, page }) => {
   await page
     .getByRole('option', { name: dummyPlace.address.municipality })
     .click();
-  await page.getByLabel('Straat en nummer').click();
+  await page.getByLabel('Straat en nummer').nth(0).click();
```

Comment:
I slightly prefer the first method

Reply:
What do you mean? 🤔

Reply:
I think the issue is maybe placing an actual field inside a field's info slot, which means the label applies to both. perhaps, I can find a way to work around it so that it's better from a semantics/a11y standpoint